statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -314,34 +481,36 @@ def score(self, params, *args, **kwargs):
         if nargs < 3:
             kwargs.setdefault('set_params', False)
 
-        initial_state = kwargs.pop('initial_state', None)
-        initial_state_cov = kwargs.pop('initial_state_cov', None)
-        if initial_state is not None and initial_state_cov is not None:
-            # If initialization is stationary, we don't want to recalculate the
-            # initial_state_cov for each new set of parameters here
-            initialization = self.initialization
-            _initial_state = self._initial_state
-            _initial_state_cov = self._initial_state_cov
-            _initial_variance = self._initial_variance
+        return approx_fprime_cs(params, self.loglike, args=args, kwargs=kwargs)
+
+    def score_obs(self, params, **kwargs):
+        """
+        Compute the score per observation, evaluated at params
 
-            self.initialize_known(initial_state, initial_state_cov)
+        Parameters
+        ----------
+        params : array_like
+            Array of parameters at which to evaluate the score.
+        *args, **kwargs
+            Additional arguments to the `loglike` method.
 
-        score = approx_fprime_cs(params, self.loglike, epsilon=1e-9, args=args,
-                                 kwargs=kwargs)
+        Returns
+        ----------
+        score : array (nobs, k_vars)
+            Score per observation, evaluated at `params`.
 
-        if initial_state is not None and initial_state_cov is not None:
-            # Reset the initialization
-            self.initialization = initialization
-            self._initial_state = _initial_state
-            self._initial_state_cov = _initial_state_cov
-            self._initial_variance = _initial_variance
+        Notes
+        -----
+        This is a numerical approximation.
```

Comment:
too vague, more precise would be to mention numerical (complex-step) derivatives